lib/app/console_app.rb

```diff
@@ -129,13 +129,13 @@ def connector_registered?(warn_if_not: true)
     end
 
     def create_connector(index_name, force: false)
-      connector_settings = Core::ConnectorSettings.fetch(connector_id)
-
-      if connector_settings.nil? || force
-        created_id = Core::ElasticConnectorActions.create_connector(index_name, App::Config['service_type'])
-        connector_settings = Core::ConnectorSettings.fetch(created_id)
+      id = connector_id
+      if Core::ElasticConnectorActions.get_connector(connector_id)[:found] == false || force
+        id = Core::ElasticConnectorActions.create_connector(index_name, App::Config['service_type'])
       end
 
+      connector_settings = Core::ConnectorSettings.fetch(id)
```

Comment:
```suggestion
      connector_settings = begin
        Core::ConnectorSettings.fetch(id)
      rescue Core::ConnectorSettings::ConnectorNotFoundError
        Core::ElasticConnectorActions.create_connector(index_name, App::Config['service_type'])
      end
```
With the above suggestion, `Core::ElasticConnectorActions.create_connector` should return connector setting object, not just `id`.

Reply:
+1 to that, but out of scope of this PR as this PR only adds human-readable messages, not changing the way CLI works.